Change quickstarts to use new secrets format (#137)
spiceai/quickstarts: postgres/README.md

**Preparation**

> - Install [Postgres](https://www.postgresql.org/download/). Upon downloading, you should be able to successfully run the following commands:

Reviewer: ```suggestion
- Install [PostgresSQL](https://www.postgresql.org/download/). Once downloaded and installed, run the following commands:
```

        pg_port: 5432
        pg_db: spice_demo
        pg_sslmode: disable
        pg_pass: ${env:PG_PASS}
```

> Ensure the `PG_PASS` environment variable is set to the password for your Postgres instance. This can be specified on the command line when running the Spice runtime, or in a `.env` file in the same directory as `spicepod.yaml`.

Reviewer: ```suggestion
Ensure the `PG_PASS` environment variable is set to the password for your Postgres instance. Environment variables can be specified on the command line when running the Spice runtime, or in a `.env` file in the same directory as `spicepod.yaml`.
```
